language/move-vm/runtime/src/loader.rs

```diff
@@ -450,9 +461,26 @@ impl Loader {
             module_cache: RwLock::new(ModuleCache::new()),
             type_cache: RwLock::new(TypeCache::new()),
             natives,
+            invalidated: RwLock::new(false),
         }
     }
 
+    /// Flush this cache if it is marked as invalidated.
+    pub(crate) fn flush_if_invalidated(&self) {
```

Comment:
better name invalidate_if_marked?

Reply:
I think the current name is right. "If the cash has been marked as invalidated, flush it now".